src/test/resources/features/Registration.feature

```diff
@@ -0,0 +1,89 @@
+    Given I am on the registration start page
+    When I click on the Start now button
+    And I click on the Submit button on the authority wizard page
+    And I select that my UK revenue is Equal to or more than £10.2 million
+    And I click on the Save and continue button
+    When I select that my AML supervisor is <AML supervisor>
+    And I click on the Save and continue button
+    Then I should be on the page with the content <Expected content>
+
+    Examples:
+      | AML supervisor              | Expected content                                                                |
+      | HMRC                        | What is your entity type?                                                       |
+      | Gambling Commission         | You need to register with the Gambling Commission (GC) to pay the levy          |
+      | Financial Conduct Authority | You need to register with the Financial Conduct Authority (FCA) to pay the levy |
+
+  Scenario Outline: User is able to select their AML supervisor as Other and proceed to the next page
+    Given I am on the AML supervisor page
+    And I click on the Submit button on the authority wizard page
+    When I select that my AML supervisor is Other
+    And I click and select the other professional body name is <Professional body name>
+    And I click on the Save and continue button
+    And I select that my entity type is <Entity type>
+    And I click on the Save and continue button
+    Then I should be on the page with the content <Expected content>
+
+    Examples:
+      | Professional body name            | Entity type                   | Expected content                                 |
+      | otherProfessionalBody__option--0  | Limited company               | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--1  | Limited liability partnership | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--5  | Limited partnership           | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--10 | Scottish limited partnership  | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--15 | General partnership           | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--20 | Scottish partnership          | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--19 | Sole trader                   | Stub GRS Journey Data                            |
+      | otherProfessionalBody__option--18 | Other                         | Sorry, we’re experiencing technical difficulties |
+
+
+  Scenario Outline: User is able to select their AML supervisor as HMRC and proceed to the next page
+    Given I am on the AML supervisor page
+    And I click on the Submit button on the authority wizard page
```

Comment:
This doesn't make sense, as the 'Given' says you are on the AML supervisor page, so the sign in should be before this?